mozilla-lockwise/lockwise-ios
# Universal Links

Fixes #[1100](https://github.com/mozilla-lockwise/lockwise-ios/issues/1100)

## Testing and Review Notes

- Run the application on an iOS device
- Open the Notes application 
- Create new note with the following universal link: https://bhqf.adj.st
- Click that link and Lockwise should open
